conf/defaults.yaml

```diff
@@ -231,16 +228,13 @@ topology.multilang.serializer: "org.apache.storm.multilang.JsonSerializer"
 topology.shellbolt.max.pending: 100
 topology.skip.missing.kryo.registrations: false
 topology.max.task.parallelism: null
-topology.max.spout.pending: null
+topology.max.spout.pending: null  # TODO: We dont need this any more
 topology.state.synchronization.timeout.secs: 60
-topology.stats.sample.rate: 0.05
+topology.stats.sample.rate: 0.001
```

Comment:
Yes I don't feel we need to change it if it's only for performance. We already know the sampling rate affects performance, and tolerate it.
Btw, likewise I commented, I think we have touched so much variables irrespective of the message subsystem and JCQueue. How you can prove introducing JCQueue brings better throughput and/or lower latency in this patch? How about message subsystem? Let's not touch variables and address them in separate issue or patch.